README.md

```diff
@@ -3,4 +3,79 @@
 ## Objectives
 + Explain what the DOM is
 + Explain the DOM hierarchy
-+ Explain how to access the DOM with JavaScript
\ No newline at end of file
++ Explain how to access the DOM with JavaScript
+
+## Intro
+Let's think about Gmail for a second. When you're looking at your inbox, there are a million actions you can take. You favorite an email and the star turns yellow without a new page loading. You add a tag to an email and the tag also appears without the page reloading. Even opening an email just replaces the inbox content with the email content without loading a new page. All of this is done using JavaScript and jQuery. But how was the code able to manipulate the HTML? By using traversing and manipulating the DOM.
```

Comment:
actually now i'm not sure if Gmail uses jQuery - might be better to just say "all of this is done using JavaScript."?